src/apb/dat/Makefile.j2

```diff
@@ -0,0 +1,21 @@
+DOCKERHOST = {{ apb_dict['dockerhost'] }}
+DOCKERORG = {{ apb_dict['app_org'] }}
+IMAGENAME = {{ apb_dict['app_name'] }}
+TAG = latest
+USER=$(shell id -u)
+PWD=$(shell pwd)
+build_and_push: apb_build docker_push apb_push
+
+.PHONY: apb_build
+apb_build:
+	docker run --rm --privileged -v $(PWD):/mnt:z -v $(HOME)/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $(USER) docker.io/ansibleplaybookbundle/apb:latest prepare
+	docker build -t $(DOCKERHOST)/$(DOCKERORG)/$(IMAGENAME):$(TAG) .
+
+.PHONY: docker_push
+docker_push:
+	docker push $(DOCKERHOST)/$(DOCKERORG)/$(IMAGENAME):$(TAG)
+
+.PHONY: apb_push
+apb_push:
+	docker run --rm --privileged -v $(PWD):/mnt:z -v $(HOME)/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $(USER) docker.io/ansibleplaybookbundle/apb:latest push
```

Comment:
@philbrookes @rthallisey 
Question: do we want the `--openshift` option here, on the push ?

Reply:
It depends what registry the broker is using.  If the broker is not using the internal openshift registry, it's not going to find the apb.  I think it's good the way it is with no assumptions.